packages/dumb/src/components/trends/cbg/cbg-slice/cbg-slices-container.tsx

```diff
@@ -0,0 +1,98 @@
+/*
+ * Copyright (c) 2016-2023, Diabeloop
+ *
+ * All rights reserved.
+ *
+ * Redistribution and use in source and binary forms, with or without
+ * modification, are permitted provided that the following conditions are met:
+ *
+ * 1. Redistributions of source code must retain the above copyright notice, this
+ *    list of conditions and the following disclaimer.
+ *
+ * 2. Redistributions in binary form must reproduce the above copyright notice,
+ *    this list of conditions and the following disclaimer in the documentation
+ *    and/or other materials provided with the distribution.
+ *
+ * THIS SOFTWARE IS PROVIDED BY THE COPYRIGHT HOLDERS AND CONTRIBUTORS "AS IS"
+ * AND ANY EXPRESS OR IMPLIED WARRANTIES, INCLUDING, BUT NOT LIMITED TO, THE
+ * IMPLIED WARRANTIES OF MERCHANTABILITY AND FITNESS FOR A PARTICULAR PURPOSE ARE
+ * DISCLAIMED. IN NO EVENT SHALL THE COPYRIGHT HOLDER OR CONTRIBUTORS BE LIABLE
+ * FOR ANY DIRECT, INDIRECT, INCIDENTAL, SPECIAL, EXEMPLARY, OR CONSEQUENTIAL
+ * DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF SUBSTITUTE GOODS OR
+ * SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS INTERRUPTION) HOWEVER
+ * CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN CONTRACT, STRICT LIABILITY,
+ * OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE) ARISING IN ANY WAY OUT OF THE USE
+ * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
+ */
+
+import React, { FunctionComponent, memo, useMemo } from 'react'
+
+import { BgBounds } from '../../../../models/blood-glucose.model'
+import { XScale } from '../../../../models/x-scale.model'
+import { CbgSliceContainerData } from '../../../../models/cbg-slice-container-data'
+import { TrendsDisplayFlags } from '../../../../models/trends-display-flags.model'
+import { CbgSliceAnimated } from './cbg-slice-animated'
+import { CbgMedianAnimated } from '../cbg-median-animated/cbg-median-animated'
+import { formatCbgs } from './cbg-slices-container.util'
+
+interface CbgSliceContainerProps {
+  bgBounds: BgBounds
+  sliceWidth: number
+  data: CbgSliceContainerData[]
+  displayFlags: TrendsDisplayFlags
+  showingCbgDateTraces: boolean
+  tooltipLeftThreshold: number
+  topMargin: number
+  xScale: XScale
+  yScale: Function
+}
+
+const CbgSliceContainer: FunctionComponent<CbgSliceContainerProps> = (props) => {
+  const {
+    bgBounds,
+    data,
+    displayFlags,
+    showingCbgDateTraces,
+    sliceWidth,
+    tooltipLeftThreshold,
+    topMargin,
+    xScale,
+    yScale
+  } = props
+
+  const cbgs = useMemo(() => {
+    return formatCbgs(data)
+  }, [data])
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🤓 nitpick:**  ‏This could be written without the explicit `return`:

```tsx
const cbgs = useMemo(() => formatCbgs(data), [data])
```